src/pykx/pandas_api/pandas_meta.py

```diff
@@ -228,6 +228,14 @@ def max(self, axis=0, skipna=True, numeric_only=False):
             res
         ), cols)
 
+    @convert_result
+    def idxmax(self, axis=0, skipna=True, numeric_only=False):
```

Comment:
The original API also accepts `index` or `columns` as possible inputs for axis. As far as I know, `preparse_computations` is unable to deal with symbols.